### ISSUE TITLE
macOS version?

### ISSUE THREAD

**Issue by jerkstore369** [originalgrego/FinalFightAE-Installer#1]
Hi, love this. Is there any way you could make a macOS version of the installer or instructions on how to do it manually as there are patching tools for macOS to do it. But then the custom version of Mame would need a port.... any chance? Thanks.

**Comment by originalgrego** [comment]
A mac/linux version of the installer is planned for later.  

You can try installing a windows copy of java through wine, then opening a terminal and launching ffae_installer.java through wine.  

Alternatively, running the combine.bat and combine_gfx.bat commands from the FFAE-Source repo in a terminal with an unzipped ffight.zip in the directory will produce two files, ffight.bin and ffight_gfx.bin.  Apply the ips patches to these files then run the split.bat and split_gfx.bat commands to produce the new rom files. Put these in a zip called ffightae.zip.

Finally, running apply_patch.bat using the source should work on linux/mac with wine installed.

**Comment by jerkstore369** [comment]
Yes! You're awesome! I'm not a programmer and am not sure how to properly setup Xcode to make a build but actually you can probably just run the patcher as is with VirtualBox and it should work. I think the bigger concern is the custom version of mame.
